### PR TITLE
Update gralde plugin for android from 2.1.2 to 2.2.0

### PR DESCRIPTION
**Pull request by hkurokawa**
# Changes
- [x] Update `build.gradle`
- [x] Disable `shrinkResource` in debug build to avoid an error (See [android - \build\intermediates\res\resources-anzhi-debug-stripped.ap_' specified for property 'resourceFile' does not exist - Stack Overflow](http://stackoverflow.com/questions/36540676/build-intermediates-res-resources-anzhi-debug-stripped-ap-specified-for-prope))
- [x] Make Lint happy

### CODE REVIEW

**Comment by konifar** [comment]
Yeah!! 🏆

**Comment by hkurokawa** [comment]
@konifar Could you review the PR? I am afraid disabling shrinkResource could be controversial. Any comments are welcome.

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java [hunk 1]
```diff
@@ -160,13 +161,19 @@ public void onBindViewHolder(BindingHolder<ItemSessionBinding> holder, int posit
                 @Override
                 public void liked(LikeButton likeButton) {
                     session.checked = true;
-                    onLikeChanged(session, position);
+                    final int pos = holder.getAdapterPosition();
+                    if (pos != RecyclerView.NO_POSITION) {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java]: Just question. Is this change necessary for gradle update?

Comment (hkurokawa) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java]: Yes. This is for making Lint happy :-) The lint complained `Do not treat position as fixed; only use immediately and call holder.getAdapterPosition() to look it up later.` and marked that as an error. So we cannot complete the compile unless it is fixed or we disable the lint.

See http://stackoverflow.com/questions/37498233/recyclerview-adapter-lint-error-do-not-treat-position-as-fixed

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java]: Got it. So strict 😮  Thanks for explanation!

File: app/build.gradle [hunk 2]
```diff
@@ -77,7 +77,6 @@ android {
             signingConfig signingConfigs.debug
             applicationIdSuffix '.debug'
             versionNameSuffix "-debug"
-            shrinkResources true
```
Comment (konifar) [inline app/build.gradle]: I think it's good. If you can check apk size, I'm appreciated. Do you have a time?

Comment (hkurokawa) [inline app/build.gradle]: Sorry, I'm on the way to the office. Will update later.

Comment (konifar) [inline app/build.gradle]: Thanks sir!

Comment (hkurokawa) [inline app/build.gradle]: Sorry for the delay in my update. The changes in the size and the number of methods are as below:

Size: 4.7MB → 5.6MB in Raw Size
droidkaigi2016_apk_vs_app-production-debug_apk_and_droidkaigi2016_apk_-_fril-android_-____git_github_com_fablic_fril-android__-_android_studio_2_2_preview_7

Number of Methods:
**Before**
droidkaigi2016_apk_-_fril-android_-____git_github_com_fablic_fril-android__-_android_studio_2_2_preview_7

**After**
app-production-debug_apk_-_fril-android_-____git_github_com_fablic_fril-android__-_android_studio_2_2_preview_7

Thanks!

Comment (konifar) [inline app/build.gradle]: Thanks for useful information.
I think it's not good but is small issue for me 😉

**Comment by konifar** [comment]
LGTM!!

**Pull request merged**